data/presentations.yaml

```diff
@@ -1,3 +1,146 @@
+- title: "Community Introductions: Ed Warnicke at Network Service Mesh"
+  link: https://www.youtube.com/watch?v=fxtG1wuLSDw&feature=youtu.be&t=17m37s
+  youtube: fxtG1wuLSDw
+  presenters: Ed Warnicke
+  organizations: Network Service Mesh
+  venue: SPIFFE Community Day
+  category: case-study
+  date: 24-Apr-2020
+  language: English
+  description: A brief overview of how the open source project Network Service Mesh is integrating SPIFFE/SPIRE
+  weight: 100
+- title: "Community Introductions: Domingo Kiser at Frontdoor"
+  link: https://www.youtube.com/watch?v=fxtG1wuLSDw&feature=youtu.be&t=14m41s
+  youtube: fxtG1wuLSDw
+  presenters: Domingo Kiser
+  organizations: Frontdoor
+  venue: SPIFFE Community Day
+  category: case-study
+  date: 24-Apr-2020
+  language: English
+  description: A brief overview of how Frontdoor is using SPIRE
+  weight: 100
+- title: "Community Introductions: Zero Trust in Health Care"
+  link: https://www.youtube.com/watch?v=fxtG1wuLSDw&feature=youtu.be&t=6m2s
+  youtube: fxtG1wuLSDw
+  presenters: Frederick Kautz, Bobby Samuel
+  organizations: doc.ai, Anthem
+  venue: SPIFFE Community Day
+  category: case-study
+  date: 24-Apr-2020
+  language: English
+  description: A brief overview of how doc.ai and Anthem are implementing zero-trust authentication using SPIRE
+  weight: 100
+- title: "Community Introductions: Eric Lee at GitHub"
+  link: https://www.youtube.com/watch?v=fxtG1wuLSDw&feature=youtu.be&t=0m45s
+  youtube: fxtG1wuLSDw
+  presenters: Eric Lee
+  organizations: GitHub
+  venue: SPIFFE Community Day
+  category: case-study
+  date: 24-Apr-2020
+  language: English
+  description: A brief overview of how GitHub is using SPIRE
+  weight: 100
+- title: SPIFFE 2020 Use Case Survey Insights
```

Comment:
I would suggest dropping this video. It's of relevance to the core community, but likely not of interest to a general user.